test/jdk/tools/jpackage/helpers-test/jdk/jpackage/test/DirectoryContentVerifierTest.java

```diff
@@ -0,0 +1,171 @@
+/*
+ * Copyright (c) 2024, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+package jdk.jpackage.test;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.Collection;
+import java.util.List;
+import java.util.Set;
+import java.util.function.BiConsumer;
+import static java.util.stream.Collectors.toSet;
+import java.util.stream.Stream;
+import jdk.jpackage.test.Annotations.Parameters;
+import jdk.jpackage.test.Annotations.Test;
+import static jdk.jpackage.test.DirectoryContentVerifierTest.AssertType.CONTAINS;
+import static jdk.jpackage.test.DirectoryContentVerifierTest.AssertType.MATCH;
+import jdk.jpackage.test.TKit.DirectoryContentVerifier;
+import static jdk.jpackage.test.TKit.assertAssert;
+
+/*
+ * @test
+ * @summary Test TKit.DirectoryContentVerifier from jpackage test library
+ * @library /test/jdk/tools/jpackage/helpers
+ * @build jdk.jpackage.test.*
+ * @modules jdk.jpackage/jdk.jpackage.internal
+ * @compile DirectoryContentVerifierTest.java
+ * @run main/othervm/timeout=360 -Xmx512m jdk.jpackage.test.Main
+ *  --jpt-run=jdk.jpackage.test.DirectoryContentVerifierTest
+ */
+public class DirectoryContentVerifierTest {
+
+    enum AssertType {
+        MATCH(DirectoryContentVerifier::match),
+        CONTAINS(DirectoryContentVerifier::contains),
+        ;
```

Comment:
Should ";" be at the end of `CONTAINS(...)` and remove ","?

Reply:
I'd prefer to keep it as is unless you strongly dislike it. The benefit is that it makes clean diffs when items added/removed from the enum.